tasks/rdma.yml

```diff
@@ -9,17 +9,15 @@
     - "libibverbs"
     - "libibverbs-utils"
     state: present
-  become: True
-  when: False
+  become: yes
 
 - name: Ensure RDMA kernel modules are loaded
   modprobe:
     name: "{{ item }}"
     state: present
-  become: True
+  become: yes
   with_items:
   - "ib_umad"
   - "ib_uverbs"
   - "rdma_ucm"
-  when: False
-
+...
```

Comment:
I don't see this file being included anywhere.

Reply:
Still not included?

Reply:
I know, I am not sure if we need it for some other configurations but this is not required for OpenHPC, hence I am reluctant to remove it.